packages/core/src/internal/utils/focus-trap.ts

```diff
@@ -125,6 +188,8 @@ export class FocusTrap {
     }
 
     this.focusTrapElement = hostElement;
+
+    console.warn('The @cds/core internal FocusTrap class is deprecated and will be removed after 5.0.');
```

Comment:
Non blocker but is there a way to not log during tests or maybe remove this and only use the `@deprecated` marker? It fills the test watcher to where it hides test failur
![Screen Shot 2021-11-02 at 5 33 32 PM](https://user-images.githubusercontent.com/2021067/139961197-4ff8f4bc-2c25-4cd2-9362-6cdd19ba6b7f.png)
es

Reply:
Yep. I can do that.

Reply:
done